site/content/docs/5.1/forms/checks-radios.md

```diff
@@ -307,6 +307,107 @@ Drop borders using `.btn-no-outline`, too.
 </div>
 {{< /example >}}
 
+<!-- Boosted mod: star rating -->
+
+## Star rating
+
+Here is an example for star rating. Simply add `.star-rating` to a `<fieldset>`, then copy the following pattern into the tags in order to have a complete star rating system. Change the `xlink:href` value to get the svg you want. You can also add as much stars as you want. The default star size is 25px (based on default font-size = 16px).
+
+{{< example >}}
+<form>
```

Comment:
Since we don't do anything with the results, I would have removed the `<form>` from the examples.